backend/stakepoold/server.go

```diff
@@ -112,6 +113,7 @@ func deriveChildAddresses(key *hdkeychain.ExtendedKey, startIndex, count uint32,
 	for i := uint32(0); i < count; i++ {
 		child, err := key.Child(startIndex + i)
 		if err == hdkeychain.ErrInvalidChild {
+			// NOTE: return slice will be shorter than requested
```

Comment:
Fair enough.  Still doesn't explain to me why their is that remote possibility, but that's OK.